script/cache_critical_dns

```diff
@@ -158,16 +158,19 @@ ensure
 end
 
 def postgres_healthcheck(host:, user:, password:, dbname:)
-  response = PG::Connection.ping(
+  client = PG::Connection.new(
     host: host,
     user: user,
     password: password,
     dbname: dbname,
     connect_timeout: 2,  # minimum
   )
-  response == PG::Constants::PQPING_OK
+  response = client.exec('select 1;')
```

Comment:
```suggestion
  response = client.exec(';')
```

We can just use an empty query here which is faster since it does not involve the PG planner.

Reply:
I've updated this to an empty query @tgxworld